game.js

```diff
@@ -299,7 +300,8 @@ class Game {
         }
 
         if (!this.ensureBotStarted(botError)) {
-            this.log("Failed to start the bot, can not make a move");
+            this.log("Failed to start the bot, can not make a move, trying to restart");
+            this.sendChat("Failed to start the bot, can not make a move, trying to restart"); // we notify user of this in ingame chat
```

Comment:
I don't think the user needs to know about this. But don't particularly mind.

Reply:
@windo 
imagine if our bot is slow to start/restart

then the user may have to wait many minutes without knowing what is going on

i think in that case, waiting say 30 seconds instead with a message after 30 seconds is much better :+1: